### PR TITLE
add CSV export for admin

### PR DESCRIPTION
**Pull request by peterberkenbosch**

### CODE REVIEW

**Comment by fastjames** [comment]
Rebased and force pushed.

**Comment by fastjames** [comment]
[ch2327]

File: app/controllers/spree/admin/subscriptions_controller.rb [hunk 1]
```diff
@@ -8,11 +8,19 @@ def index
...
+            params[:q].permit! if params[:q].present?
+            @subscriptions = @search.result(distinct: true).
+                             includes(:line_items, :user).
+                             joins(:line_items, :user).
+                             page(params[:page]).
+                             per(params[:per_page] || Spree::Config[:orders_per_page])
+          end
+          format.csv do
+            send_data(SolidusSubscriptions::DownloadService.to_csv(search: @search), filename: "subscribers.csv")
+          end
```
Comment (cnorm35) [inline app/controllers/spree/admin/subscriptions_controller.rb:22]: ❤️ service objects

**Review by cnorm35 — APPROVED** [review]

File: app/controllers/spree/admin/subscriptions_controller.rb (hunk 1 above)
Comment (peterberkenbosch) [reply app/controllers/spree/admin/subscriptions_controller.rb:22]: me too 😉

**Pull request merged**